index.bs

```diff
@@ -188,15 +194,56 @@ In addition to the Image Properties defined in [[!HEIF]], such as 'colr', 'pixi'
+In an [=AV1 Multi-Layered Image Item=], a spatial layer may improve the resolution or quality of the image decoded based on one or more of the previous layers. A layer may also provide an image that does not depend on the previous layers. Not all layers are expected to produce an image meant to be rendered. Some decoded images may be used only as intermediate encodes. The [=Sequence Header OBU=] defines the combinations of layers that are expected to produce output images. Each combination is called an [=Operating Point=]. An [=AV1 Image Item=] may contain data from multiple [=Operating Points=]. It may have [=Operating Points=] representing the same image in multiple resolutions or qualities. It may have [=Operating Points=] representing different images encoded based on a common set of layers.
 
-If a 'lsel' property is associated with an [=AV1 Image Item=], renderers shall render only the output image of the operating point whose index is equal to the 'layer_id' value and the 'ispe' property shall describe that output image. In the absence of an 'lsel' property association, the 'ispe' property shall describe the last output image and renderers are free to either render operating point outputs as they are decoded or to wait until the last operating point output image is decoded before rendering the image item.
+<h4 id ="layered-properties">Properties</h4>
+<h5 id ="layer-selector-property">Use of Layer Selector Property</h4>
+The 'lsel' property defined in [[!HEIF]] may be associated to an [=AV1 Multi-Layered Image Item=].
+
+If an [=AV1 Multi-Layered Image Item=] is associated with an 'lsel' property, the 'layer_id' value shall be between 0 and [=operating_points_cnt_minus_1=]. The 'ispe' property shall describe the dimension of the output image of the corresponding [=Operating Point=]. Renderers should use the 'layer_id' value as the return of the [=choose_operating_point=] function and should not render any other output image corresponding to other [=Operating Points=]. If the [=Operating Point=] corresponding to the 'layer_id' value exceeds the renderer's decoding capabilities, the renderer should not process the item.
+
+If an [=AV1 Multi-Layered Image Item=] is not associated with an 'lsel' property, the 'ispe' property shall describe the dimension of the output image of the operating point whose index is 0. Renderers are free to render either: only the output image of the [=Operating Point=] matching best its decoding capabilities, or to render all output images of all the [=Operating Points=] processed, resulting in a form of progressive decoding.
+
+NOTE: The dimensions of the final or intermediate output images might not match the ones given in the 'ispe' property. Renderers are expected to scale the output image to match the 'ispe' property.
```

Comment:
Why would the dimensions of the final output image not match the 'ispe' property? I can understand why the intermediates might not, but I would have expected the 'ispe' to match the final output.

Reply:
Just listened through the recording of the meeting.

If multiple outputs per operating point is possible like suggested by Wan-Teh and we let 'lsel' select a specific layer rather than operating point:
1. We need to clarify which operating point should be used if there are multiple. Or we can make it simple and say that operating point 0 shall always be used.
2. We need to clarify which operating point (if we don't always force 0) and layer the 'ispe' describes if the 'lsel' is not present

If multiple outputs per operating point is not possible, the `NOTE` above is slightly confusing. In the paragraph above it, we mention that if there is no 'lsel' the 'ispe' shall match the output of operating point 0. I think that what the note is saying is that if the decoder selects another operating point, the dimensions of the final output may not match the 'ispe' and that it should scale the image to that size if needed.

Reply:
The latest rewrite tries to address this comment.